tests/mvn_tri_test.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+
+import jax
+
+
+jax.config.update("jax_enable_x64", True)
+
+import equinox as eqx
+import jax
+import jax.numpy as jnp
+import numpy as np
+from parameterized import parameterized  # type: ignore
+
+from distreqx.distributions import MultivariateNormalTri
+
+
+class MultivariateNormalTriTest(TestCase):
```

Comment:
I would also add a test to verify all the error catching in `init` is working as expected